JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -180,7 +182,9 @@ private async Task UpdateMessageVisibilityTimeout(SQSMessage message, string rec
         {
             if (TryGetApproxReceiveCount(message.Attributes, out int approxReceiveCount))
             {
-                var visibilityTimeout = _messageBackoffStrategy.GetBackoffDuration(typedMessage, approxReceiveCount, lastException);
+                var visibilityTimeout =
+                    _messageBackoffStrategy?.GetBackoffDuration(typedMessage, approxReceiveCount, lastException) ?? TimeSpan.Zero;
```

Comment:
This is a separate behaviour change, but a little related. If a handler fails and there is no backoff strategy then it would be expected to make the message immediately visible to other workers.